README.md

```diff
@@ -96,3 +96,67 @@ instructions.
   ```
 
 3. CONTROL+C, then go back to 1
+
+### Deploy to Cloud.gov
+
+1. Initialize the database; For proof-of-concept testing, initialize with the
+`poc_data.sql` file, otherwise use the `schema.sql` file.
+
+    ```sh
+    sqlite3 aws-account-broker.db < poc_data.sql
+    ```
+
+1. Copy the `manifest.sample` file to `manaifest.yml` and edit the environment
+variables.
+1. Log in to Cloud.gov.
+
+    ```sh
+    cf login -a api.fr.cloud.gov --sso
+    ```
+
+1. For now, target your sandbox
+
+    ```sh
+    cf cf target -o <ORG> -s <SPACE>
+    ```
+
+1. Push the app.
+
+    ```sh
+    cf push --random-route aws-account-broker
+    ```
+
+1. In the Cloud.gov dashboard, navigate to the app and set the following
+environment variables:
+
+    - `BASE_EMAIL`
+    - `BROKER_USER`
+    - `BROKER_PASSWORD`
```

Comment:
Just FYI that you can also [set these from the command line](http://cli.cloudfoundry.org/en-US/cf/set-env.html).